RELEASE/scripts/autoscend/quests/level_04.ash

```diff
@@ -65,7 +65,7 @@ boolean L4_batCave()
 			return false;
 		}
 
-		buffMaintain($effect[Polka of Plenty], 15, 1, 1);
+		provideMeat(1000, $location[The Boss Bat\'s Lair], false);
```

Comment:
1000 is a lot. Wouldn't be good to use too many items to buff meat for this, like a battery for instance. We're just trying to get a bit of extra value from the bats. Maybe set goal to like 50? Or perhaps move more items to only be used if doEverything is true. So only use items we typically have many of for bats

Reply:
Good call. 60% is just from the unlimited buffs (Disco Leer and Polka of Plenty) so that's a good idea